src/elements/ActionMenuItem/actionsMenuItem.scss

```diff
@@ -37,13 +39,23 @@
       color: $spunPearl;
       cursor: not-allowed;
 
-      .action-menu__icon {
+      .actions-menu__icon {
         svg {
           path:first-child {
             fill: rgba($black, 0.2);
           }
         }
       }
     }
+
+    &_delete {
+      color: $amaranth;
+
+      svg {
+        path:first-child {
+          fill: $amaranth;
+        }
+      }
+    }
```

Comment:
It is not a good practice to identify specific actions within a common component. The component is used throughout the platform. It is better to have predefined styles that can be used for various actions instead of just one. This approach is similar to how Bootstrap has styles such as 'primary', 'warning', and 'danger'.